js/common/util/compute_domain.js

```diff
@@ -0,0 +1,6 @@
+export default function computeDomain(stateObj) {
+  if (!stateObj || !stateObj.entity_id) {
```

Comment:
Let's not do this. It's always better to fail hard and let people calling our function deal with it. It's also not in the old function.